docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github/index.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_label: GitHub
+---
+
+# The GitHub Webhook Receiver
+
+The following instructions will work for Github,
+Github Enterprise Cloud, and GitHub Enterprise Server.
+
+## Configuring the Receiver
+
+The Kargo GitHub webhook receiver will require a Kubernetes Secret. This Secret is required to contain a `secret` key in its data map. You will be required to provide the value assigned to the `secret` key to GitHub in a later step so keep it handy.
+
+:::note
+The following command can be used to generate a sufficiently secure secret:
+
+```shell
+echo "$(openssl rand -base64 48 | tr -d '=+/' | head -c 32)"
+```
+
+:::
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gh-wh-secret
+  namespace: kargo-demo
+stringData:
+  secret: your-secret-here
+---
+apiVersion: kargo.akuity.io/v1alpha1
+kind: ProjectConfig
+metadata:
+  name: kargo-demo
+  namespace: kargo-demo
+spec:
+  webhookReceivers: 
+    - name: gh-wh-receiver
+      github:
+        secretRef:
+          name: gh-wh-secret
+```
+
+Github will sign payloads using this `secret` via HMAC signature. Our webhook receiver will use this secret to verify the signature.
```

Comment:
I would still move this higher. Tell users what the secret's for exactly before showing them how to generate.